### PR TITLE
Support pulling of single locales

### LINKED ISSUE #185: Option to pull translations of specific locales
I have a project where we use same set of translations on different projects/variants.

Currently pulling translations pulls translations from all locales. There should be option to limit the locales to be pulled.

### PR DESCRIPTION
**Pull request by full-of-foo**
Resolves #185

### CODE REVIEW

File: lib/localeapp/cli/pull.rb [hunk 1]
```diff
@@ -2,10 +2,13 @@ module Localeapp
...
+      @locale_key = nil
 
-      def execute
+      def execute(locale_key = nil)
         @output.puts "Localeapp Pull"
+        @output.puts "Fetching #{locale_key}" if locale_key
         @output.puts ""
+        @locale_key = locale_key
 
         @output.puts "Fetching translations:"
```
Comment (tjouan) [inline lib/localeapp/cli/pull.rb]: See my comment about merging messages in the UAT scenario.

Then I would favor assigning instance variable first, then writing to
standard output, which I think we could refactor with only one `#puts`
call.

File: lib/localeapp/cli/pull.rb [hunk 2]
```diff
@@ -26,6 +29,17 @@ def report_failure(response)
         @output.puts "Failed!"
         fail APIResponseError, "API returned #{response.code} status code"
       end
+
+      private
+
+      def yaml_data(response)
```
Comment (tjouan) [inline lib/localeapp/cli/pull.rb]: I think that a class method in `Localeapp` (like `Localeapp.load_yaml`), or even a new dedicated class/module would be better:
* I don't think it's `Localeapp::CLI::Pull` responsibility;
* I don't see an easy way to unit-test this code here.

**Comment by tjouan** [comment]
@full-of-foo thank you for the PR update, could you please rebase and squash your changes? Then I will merge.

**Comment by full-of-foo** [comment]
Thanks @tjouan! Done and done.

**Comment by tjouan** [comment]
@full-of-foo thank you for the contribution!

**Comment by tjouan** [comment]
@full-of-foo FYI we just released localeapp 2.4, this new feature is included.

**Pull request merged**